entries/swc-136.md

```diff
@@ -0,0 +1,16 @@
+# Title
+Unencrypted Private Data On-Chain
+
+## Relationships
+[CWE-767: Access to Critical Private Variable via Public Method](https://cwe.mitre.org/data/definitions/767.html)
+
+## Description
+It is a common misconception that `private` type variables cannot be read. Even if your contract is not open source, attackers can look at contract transactions to determine values stored in the state of the contract. For this reason, it's important that unencrypted private data is not stored in the contract code or state.
+
+## Remediation
+Any private data should either be stored off-chain, or carefully encrypted.
```

Comment:
Are there any projects that encrypt data and store it on-chain and if so can we add references below?